CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 ## Unreleased
 
+### Migration notes
+
+### 🌟 New javaagent instrumentation
+
+- Add instrumentation for vertx-sql-client
+  ([#8311](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8311))
+
+### 🌟 New library instrumentation
+
+### 📈 Enhancements
+
+- Make net.transport an optional attribute
+  ([#8279](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8279))
+- AddingSpanAttributes annotation
```

Comment:
Is that the name or there's a space missing between `Adding` and `SpanAttributes`?

Reply:
That's the literal name of the attribute [@AddingSpanAttributes](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation-annotations/src/main/java/io/opentelemetry/instrumentation/annotations/AddingSpanAttributes.java#L35). It's a new feature that allows one to annotate a method to add attributes to existing parent context without creating a new span.

Reply:
thx! I updated this to make it clearer